server.js

```diff
@@ -7,5 +8,9 @@ const PORT = process.env.PORT || 3000;
 app.use(express.json());
 app.use(routes);
 app.listen(PORT, () => console.log(`server running on port ${PORT}`));
+app.use((req, res, next) => {
+  if (!req.route) { return res.status(400).json({ status: 400, error: 'Incorrect Route' }); }

```

Comment:
format this block so that it looks good. Ensure that your Eslint is working well.